### ISSUE TITLE
Error when parsing response: Expected param start-info="application/soap+xml", got text/xml

### ISSUE THREAD

**Issue by giovanni-orciuolo** [hooklift/gowsdl#229]
The SOAP service I'm communicating with responds in MTOM mode. Sample response:

```
--uuid:f1eb0f5f-52d7-416d-8bf1-02e8b7b7036e
Content-Type: application/xop+xml; charset=UTF-8; type="text/xml";
Content-Transfer-Encoding: binary
Content-ID: 
<root.message@cxf.apache.org>
    <soap:Envelope xmlns:soap="http://schemas.xmlsoap.org/soap/envelope/">
        <soap:Body>
            omitted for brevity
        </soap:Body>
    </soap:Envelope>
--uuid:f1eb0f5f-52d7-416d-8bf1-02e8b7b7036e--
```

I get this error when I try to parse this response through the soap library:
`Expected param start-info="application/soap+xml", got text/xml`

Any help? I can't tweak the response I get from this bs service

**Comment by giovanni-orciuolo** [comment]
I just opened a PR to solve this issue: #230

**Comment by giovanni-orciuolo** [comment]
Fixed in PR #230
